py-polars/tests/unit/operations/arithmetic/test_list.py

```diff
@@ -0,0 +1,26 @@
+import polars as pl
+
+
+def test_literal_broadcast_list() -> None:
```

Comment:
You should also test eager operations in addition to lazy operations, e.g. `df.get_column("A") + lit`, it hits different code paths in this case so you get different bugs.